pkg/config/config.go

```diff
@@ -52,6 +53,17 @@ type BackupS3 struct {
 	Region  string `yaml:"region"`
 }
 
+type BackupScp struct {
+	Host       string `yaml:"host"`
+	Port       string `yaml:"port"`
+	User       string `yaml:"user"`
+	Path       string `yaml:"path"`
+	SshKey     string `yaml:"sshKey"`
+	PassPhrase string `yaml:"passPhrase"`
+	Password   string `yaml:"password"`
+	Key        string `yaml:"key"`
```

Comment:
Is it possible to merge `Key`  and `RemotePath` field?

Reply:
It is certainly better that way, so we will integrate it!